src/renderer/components/home.vue

```diff
@@ -1,37 +1,43 @@
 <template>
-    <div class="grid-container fluid text-center">
-        <div class="grid-x grid-padding-y">
-          <div id="challenge" class="cell small-2"></div>
-          <div class="cell small-8">
-              <h2>Tournament Management System</h2>
-              <h4>v2018.2.0</h4>
-          </div>
-          <div id="fll" class="cell small-2"></div>
+    <div class="ui center aligned grid container">
+        <div class="row">
+            <div id="challenge" class="three wide column"></div>
+            <div class="ten wide column">
+                <h2>Tournament Management System</h2>
+                <h4>v2019.0.0</h4>
+            </div>
+            <div id="fll" class="three wide column"></div>
         </div>
-        <div class="grid-y">
-          <div class="cell grid-x grid-margin-x">
-            <fieldset v-for="module in modules" class="cell small-3 fieldset">
-              <legend class="text-left">
-                <a @click="event => openSite(event, module.site)" :href="module.site">{{module.name}}</a>
-              </legend>
-              <label>
-                <div><qrcode :value="module.site"></qrcode></div>
-                <a @click="event => openSite(event, module.site)" :href="module.site">{{module.site}}</a>
-                <button class="button" @click="() => saveInClipboard(module.site)"><i class="fas fa-copy"></i>&nbsp;Copy link</button>
-              </label>
-            </fieldset>
-          </div>
+        <div class="row ui four cards">
+            <div v-for="module in modules" class="card">
+                <div class="content">
+                    <a class="header" @click="event => openSite(event, module.site)" :href="module.site">
+                        {{module.name}}
+                    </a>
+                    <div class="ui divider"></div>
+                    <div class="description">
+                        <qrcode :value="module.site"></qrcode>
+                    </div>
+                </div>
+                <div class="extra content">
+                    <a class="header" @click="event => openSite(event, module.site)" :href="module.site">
```

Comment:
This header has no content, therefore it's not really a header. It should be a simple link, and the button should be on it's right.